src/Templates/Boilerplate/Bit.Boilerplate/.docs/15- Logging, OpenTelemetry and Health Checks.md

```
@@ -7,7 +7,7 @@ Welcome to Stage 15! In this stage, you'll learn about the comprehensive logging
 ## Table of Contents
 
 1. [ILogger for Errors, Warnings, and Information](#ilogger-for-errors-warnings-and-information)
-2. [Activity and AppActivitySource for Tracking Operations](#activity-and-appactivitysource-for-tracking-operations)
+2. [Activity and Meter for Tracking Operations](#activity-and-meter-for-tracking-operations)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)